### PR TITLE
extend recipe for replacing deprecated management.* properties in Spring Boot 3

### PR DESCRIPTION
**Pull request by iuliiasobolevska**

### CODE REVIEW

**Comment by timtebeek** [comment]
Really appreciate you providing these fixes here! It's already a big improvement as it is now; one small question above before we merge.

**Review by timtebeek — APPROVED** [review]

File: src/main/resources/META-INF/rewrite/spring-boot-30.yml [hunk 1]
```diff
@@ -819,31 +840,49 @@ recipeList:
...
   - org.openrewrite.java.spring.ChangeSpringPropertyKey:
       oldPropertyKey: management.metrics.export.wavefront.step
       newPropertyKey: management.wavefront.metrics.export.step
   - org.openrewrite.java.spring.ChangeSpringPropertyKey:
       oldPropertyKey: management.metrics.export.wavefront.uri
-      newPropertyKey: management.wavefront.uri
+      newPropertyKey: management.wavefront.metrics.export.uri
+  - org.openrewrite.java.spring.ChangeSpringPropertyKey:
+      oldPropertyKey: management.trace.http.enabled
+      newPropertyKey: management.httpexchanges.recording.enabled
```
[the remark this answers is not in the document]
Comment (timtebeek) [reply src/main/resources/META-INF/rewrite/spring-boot-30.yml]: As mentioned in https://github.com/openrewrite/rewrite-spring/pull/356#discussion_r1200413413 we're merging all except this one change early, such that we can already benefit from these changes. Optional to track that one property separately, or as part of #357

[the remark this answers is not in the document]
Comment (iuliiasobolevska) [reply src/main/resources/META-INF/rewrite/spring-boot-30.yml]: Apologies for the delayed response!

I got `management.httpexchanges.recording.enabled` from:
* https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.http-exchanges.custom
* and https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0.0-RC2-Configuration-Changelog 
* and it's been referenced here: https://github.com/spring-projects/spring-boot/blob/9643dbeed26268f37d37c22385704de8911b8f21/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/web/exchanges/HttpExchangesAutoConfiguration.java#LL43C1-L43C1

[the remark this answers is not in the document]
Comment (timtebeek) [reply src/main/resources/META-INF/rewrite/spring-boot-30.yml]: Thanks for the links! I'd been getting zero search hits when I tried to find a source; I'll apply that missing migration, you were absolutely right! :)

[the remark this answers is not in the document]
Comment (iuliiasobolevska) [reply src/main/resources/META-INF/rewrite/spring-boot-30.yml]: thank you! appreciate it!

**Pull request merged**